### PR TITLE
Uiupdate

### PR DESCRIPTION
**Pull request by hsachinraj**

### CODE REVIEW

File: labs/vstsextend/Jenkins/README.md [hunk 1]
```diff
@@ -30,343 +28,250 @@ While there are pros and cons with both the approaches, the latter approach has
...
+Next, you will configure a Azure CD pipelines to fetch and deploy the artifacts produced by the build. Since the deployment is being done to Azure, an endpoint to Azure will be configured. An endpoint to Jenkins server will also be configured, if not configured earlier.
 
-1. After the endpoint creation, click on the **Build & Release** hub and then select the **Releases** section. Click on the **+ Create a new Release definition** button to initiate a new release definition
+1. After the endpoint creation, go to the **Releases** tab in **Azure Pipelines**. Open the + drop-down in the list of release pipelines, and choose **Create release pipeline**.
 
-1. Since a web application will be published to Azure, the **Azure App Service Deployment** template will be used for the configuration.
+1. Select the **Azure App Service Deployment** template. 
 
-   ![New Release Definition](images/newreleasedefintion.png)
+   <img class="myImg" src="images/newreleasedefintion.png" alt="New Release Definition" />
```
Comment (Hoyg) [inline labs/vstsextend/Jenkins/README.md]: Old Screenshot.  To be replaced with new UI screenshot

File: labs/vstsextend/Jenkins/README.md [hunk 2]
```diff
@@ -30,343 +28,250 @@ While there are pros and cons with both the approaches, the latter approach has
...
-    ![Adding Credentials to Jenkins](images/jenkinscredentials.png)
+    <img class="myImg" src="images/jenkinscredentials.png" alt="Adding Credentials to Jenkins"/>
 
 1. Select the credential created in the previous step from the drop-down. The error message should disappear now.
 
-   ![VSTS Git config in Jenkins](images/jenksaddvstsgit.png)
+     <img class="myImg" src="images/jenksaddvstsgit.png" alt="VSTS Git config in Jenkins"/>
 
-   >The source code for this application has both unit tests and UI tests. Only the unit tests will be included in this lab.
+1. The source code for this application has both unit tests and UI tests. We are not ready to run the UI test at this point. So, we will specify to run only the unit tests. Scroll down to the **Build** section and provide the text `package -Dtest=FaresTest,SimpleTest` in the **Goals and options** field. Select **Save** to save and navigate to the home page of the project.
```
[the remark this answers is not in the document]
Comment (hsachinraj) [reply labs/vstsextend/Jenkins/README.md]: Removed it. Thanks

File: labs/vstsextend/Jenkins/README.md [hunk 3]
```diff
@@ -30,343 +28,250 @@ While there are pros and cons with both the approaches, the latter approach has
...
+   `http://dev.azure.com/{your org name}{team project name}/_git/MyShuttleDocker`. If you do not see **Git** under Source code management (not a usual thing), you will need to install/enable the Git plugin.
 
-   The VSTS Git repos are private and requires user credentials to be provided to access the repository. If the Git credentials is not set already, it can be done from the VSTS.
+   <img class="myImg" src="images/jenkins-vstsrepo.png" alt="Configuring Azure DevOps Git URL"/>
 
-1. Go to your VSTS project page and select the **Clone** option, provide the `User name` and `Password` and then click on the **OK** button.
+1. Your Azure repo is very likely to be private. Unless you have public repo, you should provide the credentials to access the repository. If you do not have one or don't remember the credentials, go to your Azure Repos and select the **Clone** option. Select **Generate Credentials** and enter a `User name` and `Password`. Click **OK** to save and exit.
 
-   ![Generating Git Credentials](images/vsts-generategitcreds.png)
+   <img class="myImg" src="images/vsts-generategitcreds.png" alt="Generating Git Credentials"/>
```
[the remark this answers is not in the document]
Comment (hsachinraj) [reply labs/vstsextend/Jenkins/README.md]: Replaced existing screenshot with a new one

File: labs/vstsextend/Jenkins/README.md [hunk 4]
```diff
@@ -30,343 +28,250 @@ While there are pros and cons with both the approaches, the latter approach has
...
-1. The **Jenkins Download Artifacts** task will download the build artifacts from the Jenkins job to the staging directory
 
-    ![Download Jenkins Artifact](images/downloadjenkinsartifact.png)
+1. The **Jenkins Download Artifacts** task will download the build artifacts from the Jenkins job to the staging directory
 
-1. The **Publish Artifact drop** will publish to the VSTS.
+    <img class="myImg" src="images/downloadjenkinsartifact.png" alt="Download Jenkins Artifact" />
 
-1. Click on the **Save & queue** button to complete the build definition configuration and initiate a new build.
+1. The **Publish Artifact drop** will publish to Azure Pipelines.
```
[the remark this answers is not in the document]
Comment (hsachinraj) [reply labs/vstsextend/Jenkins/README.md]: Nope! it should be Azure Pipelines only - see https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-build-artifacts?view=vsts

File: labs/vstsextend/Jenkins/README.md [hunk 5]
```diff
@@ -30,343 +28,250 @@ While there are pros and cons with both the approaches, the latter approach has
...
 
-## Deploying Jenkins Artifacts with Release Management
+1. Click **Save & queue** button to save and initiate a new build.
 
-The next step is to configure the VSTS Release Management to fetch and deploy the artifacts produced by the build.
+## Linking the build artifact for deployment in a CD pipeline    
 
-1. Since the deployment is being done to Azure, an endpoint to Azure will be configured. An endpoint to Jenkins server will also be configured, if not configured earlier.
+Next, you will configure a Azure CD pipelines to fetch and deploy the artifacts produced by the build. Since the deployment is being done to Azure, an endpoint to Azure will be configured. An endpoint to Jenkins server will also be configured, if not configured earlier.
```
[the remark this answers is not in the document]
Comment (hsachinraj) [reply labs/vstsextend/Jenkins/README.md]: Corrected

**Pull request merged**